### PR TITLE
VACMS-6396: Add EFF fields to 3 top task pages.

### LINKED ISSUE #6396: Add EFF fields to 3 top task pages
## User Story or Problem Statement

As an editor of a top task page, i can see the nationalized content inline within the editorial experience, so that i can craft my localized content accordingly. 

As a FE developer, i can access nationalized content via a subquery of a top task page, so that i can build a template.

## Acceptance Criteria
- [ ] All centralized content fields on Register for care, Billing and insurance, Medical records have their EFF fields on their respective content types. 

## Implementation steps

Add `field_cc_[fieldname]*` fields to content types, with UUIDs. 

## Design principles

Which [CMS design principle](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/cms/cms-product-design-principles.md) is at play?
- [ ] Purpose-driven: Creates an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [ ] Efficient: Removes distractions and create clear, straightforward paths to get the job done.
- [ ] Approachable: Offers friendly guidance over authoritative instruction.
- [ ] Consistent: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [x] Empowering: Provide clear information to help editors make decisions about their work.

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [ ] `Core Application Team`
- [x] `Product Support Team`

### PR DESCRIPTION
**Pull request by kevwalsh**
## Description

Closes #6396. 

## QA steps

- [ ] validate /node/add/vamc_system_billing_insurance EFF content is connected
- [ ] validate /node/add/vamc_system_medical_records_offi EFF content is connected
- [ ] validate /node/add/vamc_system_register_for_care EFF content is connected

### Definition of Done

- [ ] Documentation has been updated, if applicable.
- [ ] Tests have been added if necessary.
- [ ] Automated tests have passed.
- [ ] Code Quality Tests have passed.
- [ ] Acceptance Criteria in related issue are met.
- [ ] Manual Code Review Approved.

### Select Team for PR review

- [ ] `Core Application Team`
- [x] `Product Support Team`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing annoucement

Is an announcement needed to let editors know of this change? 
- [ ] Yes, and it's written in issue ____ and queued for publication. 
  - [ ] Merge and ping @ rachel-kauff so she's ready to publish after deployment
- [ ] Yes, but it hasn't yet been written 
  - [ ] Don't merge yet -- ping @ rachel-kauff to prompt her to write and queue content
- [ ] No announcement is needed for this code change. 
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

**Comment by kevwalsh** [comment]
@swirtSJW looks good. We're going to want to split some of those EFF fields into separate Tooltip fieldgroups, but that can wait until we have all the other fields present too. 

You'll also want to restore the revision log placement, although i think there's an automated test for that.

File: config/sync/core.entity_form_display.node.vamc_system_billing_insurance.default.yml [hunk 1]
```diff
@@ -12,15 +15,15 @@ dependencies:
...
     - allow_only_one
     - content_moderation
+    - entity_field_fetch
     - field_group
 third_party_settings:
   field_group:
     group_edit:
       children:
         - moderation_state
-        - revision_log
```
Comment (swirtSJW) [inline config/sync/core.entity_form_display.node.vamc_system_billing_insurance.default.yml:21]: grr

**Comment by swirtSJW** [comment]
@kevwalsh Yah I thought about splitting them out into separate tooltip groups but it didnt makes sense without knowing what other editable fields will be there.  So I lumped them together for now.  Splitting them out will need no change to tests, so its an easy task.
 I fixed the awol revision logs too.

**Review by ethanteague — DISMISSED** [review]
LGTM

**Pull request merged**